documentation/dev/api/swagger.json

```diff
@@ -0,0 +1,123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "prompb/rpc.proto",
+    "version": "version not set"
+  },
+  "schemes": [
+    "http",
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/admin/v1/tsdb/delete_series": {
```

Comment:
Yea, we've to make a call. Either we make it `/v2/{admin,...}` or we declare the admin API as it's own thing, in which case it starts with that as a prefix and is v1 `/admin/v1/...`.

Opinions @Gouthamve @juliusv